lib/utils.ts

```diff
@@ -35,3 +35,7 @@ export function getResizedImgUrl(
         targetWidth
     : "";
 }
+
+export function isOwnerEmailDisplayed(input: string): boolean {
+  return /^anezka@cesko.digital|^gabriela@cesko.digital/.test(input)
```

Comment:
Nechcem tady spíš něco jako `"".endsWith("cesko.digital")`?

Reply:
> Nechcem tady spíš něco jako `input.endsWith("cesko.digital")`?
Jen pro doplnění, finálně určitě ano, jen v této verzi zatím omezujeme na konkrétní emaily.